### ISSUE TITLE
Is there a working fork of this repo?

### ISSUE THREAD

**Issue by tamsky** [xdissent/ievms#341]
It would appear this repo is idle or abandoned.
***If you know of a working version/fork of this repo please point it out to everyone.***

Currently this script doesn't work, and none of the forks work either.

I see more than a small amount of refactoring ahead to make the current download URLs (via https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/)::

```
# IE8 on win7
# https://az792536.vo.msecnd.net/vms/VMBuild_20150916/VirtualBox/IE8/IE8.Win7.VirtualBox.zip
# IE9 on win7
# https://az792536.vo.msecnd.net/vms/VMBuild_20180306/VirtualBox/IE9/IE9.Win7.VirtualBox.zip
# IE10 on win7
# https://az792536.vo.msecnd.net/vms/VMBuild_20150916/VirtualBox/IE10/IE10.Win7.VirtualBox.zip
# IE11 on win7
# https://az792536.vo.msecnd.net/vms/VMBuild_20180102/VirtualBox/IE11/IE11.Win7.VirtualBox.zip
# md5: 48f7ab9070c7703cf50634479b8ead38
# IE11 on Win81
# https://az792536.vo.msecnd.net/vms/VMBuild_20180102/VirtualBox/IE11/IE11.Win81.VirtualBox.zip
# MSEDGE on Win10 (x64) Stable (16.16299)
# https://az792536.vo.msecnd.net/vms/VMBuild_20171019/VirtualBox/MSEdge/MSEdge.Win10.VirtualBox.zip
# MSEDGE on Win10 (x86) Preview (17.17074)
# https://az792536.vo.msecnd.net/vms/VMBuild_20180102/VirtualBox/MSEdge/MSEdge.Win10_preview.VirtualBox.zip
```

work with the existing download code
```
    if [ "${os}" == "Win10" ]
    then
        url="https://az792536.vo.msecnd.net/vms/VMBuild_20171019/VirtualBox/MSEdge/MSEdge.Win10.
VirtualBox.zip"
    else
        url="https://az412801.vo.msecnd.net/vhd/VMBuild_20141027/VirtualBox/IE${version}/Windows
/IE${version}.${os}.For.Windows.VirtualBox.zip"
    fi
```

At which point, I'm forced to wonder if now isn't a good time to refactor the whole script.

**Comment by PeterBowater** [comment]
Indeed, broken here when trying to run, was working until quite recently, but MS seems to have moved the download locations, I get this response when trying to run:

`Downloading OVA ZIP from https://az412801.vo.msecnd.net/vhd/IEKitV1_Final/VirtualBox/OSX/IE9_Win7.zip to IE9_Win7.zip (attempt 1 of 3)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   223  100   223    0     0    369      0 --:--:-- --:--:-- --:--:--   369
MD5 check failed for IE9_Win7.zip (wanted 58d201fe7dc7e890ad645412264f2a2c, got 5d276fd55a8f7154421d4ce4721187bf)`

Happens for all images/ISOs.

Thanks for the work to date, I use this a LOT for cross-browser stuff on my Mac.

Peter.

**Comment by PeterBowater** [comment]
Looking more at this site:

https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/

URLs appear to be:

https://az792536.vo.msecnd.net/vms/VMBuild_20150916/VirtualBox/IE8/IE8.Win7.VirtualBox.zip
https://az792536.vo.msecnd.net/vms/VMBuild_20180306/VirtualBox/IE9/IE9.Win7.VirtualBox.zip
https://az792536.vo.msecnd.net/vms/VMBuild_20150916/VirtualBox/IE10/IE10.Win7.VirtualBox.zip
https://az792536.vo.msecnd.net/vms/VMBuild_20180102/VirtualBox/IE11/IE11.Win7.VirtualBox.zip
https://az792536.vo.msecnd.net/vms/VMBuild_20171019/VirtualBox/MSEdge/MSEdge.Win10.VirtualBox.zip

If I get time I will try and fork...

**Comment by heddn** [comment]
randall-k/ievms seems to work fine for EDGE

**Comment by randall-k** [comment]
Yeah, I've only had time to fix it for EDGE, haven't had time to do the others yet but I hope to do at least IE 11 as well.

**Comment by Steague** [comment]
@randall-k please just for IE11! I beg of thee!

**Comment by amichaelparker** [comment]
I forked this [here](https://github.com/amichaelparker/ievms) and fixed the download links and md5 information. All of the VM installs *should* work.

I also refactored a separate version of the script that uses Node to hit Microsoft's API to grab both the most current download link and MD5. Hopefully that version will be a little more futureproof, as the normal version has everything hard-coded (which is why this repo hasn't been working).

I've only tested it on my Macbook, but theoretically it should work on anything the original script would have worked on.

**Comment by philfreo** [comment]
@amichaelparker Thanks for this! Is this a fork you intend to maintain? If so, since this project / @xdissent seems like it's not going to be updated, perhaps you want to allow Issues on your fork? It'd be nice if the community focused on a single fork. It'd also be nice to get your version setup with `brew` etc.

**Comment by TylerTemp** [comment]
> I forked this [here](https://github.com/amichaelparker/ievms) and fixed the download links and md5 information. All of the VM installs _should_ work.
> 
> I also refactored a separate version of the script that uses Node to hit Microsoft's API to grab both the most current download link and MD5. Hopefully that version will be a little more futureproof, as the normal version has everything hard-coded (which is why this repo hasn't been working).
> 
> I've only tested it on my Macbook, but theoretically it should work on anything the original script would have worked on.

Down too.

Link <https://az792536.vo.msecnd.net/vms/VMBuild_20150916/VirtualBox/IE8/IE8.Win7.VirtualBox.zip>

gives 

```xml
<Error>
<Code>BlobNotFound</Code>
<Message>
The specified blob does not exist. RequestId:012c1c2c-901e-00b3-5d75-f8eb9a000000 Time:2020-03-12T13:55:54.2712404Z
</Message>
</Error>
```

Any other downloadable link? Thanks!

**Comment by PeterBowater** [comment]
Quite, same problem recently. Weirdly though, something seems to have nixed doing a manual installation of W10 in my Mac (in Virtualbox)  as well...

**Comment by randallbruder** [comment]
@TylerTemp 

> Link https://az792536.vo.msecnd.net/vms/VMBuild_20150916/VirtualBox/IE8/IE8.Win7.VirtualBox.zip
> gives
> ```
> <Error>
> <Code>BlobNotFound</Code>
> <Message>
> The specified blob does not exist. RequestId:012c1c2c-901e-00b3-5d75-f8eb9a000000 Time:2020-03-12T13:55:54.2712404Z
> </Message>
> </Error>
> ```
> 
> Any other downloadable link? Thanks!

That VM can be downloaded from https://archive.org/details/ie8.win7.virtualbox

Provided by @Alhadis in https://gist.github.com/zmwangx/e728c56f428bc703c6f6#gistcomment-3115797
